utils/check_publications_status.py

```diff
@@ -70,15 +94,19 @@ def upload_results(syn, results, parent):
 
 def main():
     """Main function."""
+    sys.path.insert(0, './../annotations')
```

Comment:
```suggestion
    sys.path.insert(0, './annotations')
```

The automated workflow will be running this script from the parent folder rather than from `mc2-center-dcc/utils`, so we just need to update this path a little bit!